### DISCUSSION TITLE
'There is nothing to scrape'

### DISCUSSION THREAD

**Discussion by ChrisKyle698** [UltimaHoarder/UltimaScraper#825] [Q&A]
I am having this issue :
![error scraper](https://user-images.githubusercontent.com/79023121/107958933-01792100-6fa3-11eb-811d-b242c4a6bbdb.PNG)

it goes to 10/10 but says there is nothing to scrape. I am changing the username in the auth.json file constantly to the name of the onlyfans model but it says there is nothing to scrape and it changes the username part of the auth.json file automatically back to 'default'
what should I do? I already tried re installing it

**Comment by neoplansc** [comment]
Im having the exact same problem. Have you found a solution?

**Reply by kingkarl2** [reply]
Same issue, any help yet?

**Reply by edmundackah** [reply]
Tried that still not working

**Reply by thepetcow** [reply]
I'm having the same issue with the newest version (since the old versions stopped working). No auth errors or anything, just nothing to scrape.

Anyone else have any luck? I'm editing the file through NotePad++, and I've never had issues with the older versions.

**Reply by edmundackah** [reply]
Had to redownload and use the code writer app from the windows store to edit the files

**Reply by tgbgb** [reply]
Same issue for me. Started a few days ago

**Reply by tgbgb** [reply]
> Same issue for me. Started a few days ago

![image](https://user-images.githubusercontent.com/10448931/114685077-73eb5080-9d09-11eb-9687-ff581a50321c.png)

**Comment by WillScott73** [comment]
I suspect that they are tying the session token with the useragent string. I''ve had this issue several times, and updating the session cookie alone didn't work. I realised that my browser had been updated, got the useragent string it presents, and modified my config with both the updated session token and the updated useragent string, and all was fine.

**Reply by tgbgb** [reply]
That was the first thing I tried. No joy.

**Comment by silverglade303** [comment]
I also have the same issue, I have downloaded the latest version and set it up as new. 
![image](https://user-images.githubusercontent.com/20000708/115878059-f7d5c480-a3fc-11eb-83c7-2f4288cdefb6.png)

**Comment by fightcancer2** [comment]
Issue resolved by installing the latest version.

**Comment by LunarPenguin** [comment]
@ChrisKyle698 please close / mark this discussion as answered if the problem has been solved
